### PR TITLE
Translations normalization

### PR DESCRIPTION
**Pull request by manueldevjour**
Work In Progress for translation normalization of [Devographics locale es-ES repository for StateOfJS surveys](https://github.com/Devographics/locale-es-ES)

### CODE REVIEW

**Comment by manueldevjour** [comment]
As contributors of this repository, could you check it before merging it, please? @fransyrcc @timbergus (Just pinging both of you because I saw you have recent contributions on GitHub).

File: common.yml [hunk 1]
```diff
@@ -243,31 +243,31 @@ translations:
...
-    t: Lo conozco
+    t: ✅ Lo conozco
   - key: options.features.used.short
-    t: Lo he usado
+    t: 👍 Lo he usado
 
   # patterns
   - key: options.patterns.use_never
-    t: Casi siempre lo evito
+    t: Nunca lo uso
```
Comment (timbergus) [inline common.yml:262]: Aquí hubiese puesto "No lo uso nunca".

**Review by timbergus — COMMENTED** [review]
Me parece bien. Perdona que no lo haya mirado antes. No me lo ha notificado 🙁

File: common.yml (hunk 1 above)
Comment (manueldevjour) [reply common.yml:262]: Perfecto, lo cambiaré para el próximo commit que haga con más cambios. Muchas gracias! :)

**Pull request merged**